src/bridge/MatrixHandler.ts

```diff
@@ -969,23 +950,17 @@ export class MatrixHandler {
     private async sendIrcAction(req: BridgeRequest, ircRoom: IrcRoom, ircClient: BridgedClient, ircAction: IrcAction,
         event: MatrixMessageEvent) {
         // Send the action as is if it is not a text message
-        let text = event.content.body;
-        if (event.content.msgtype !== "m.text" || !text) {
+        if (event.content.msgtype !== "m.text" || !event.content.body) {
             await this.ircBridge.sendIrcAction(ircRoom, ircClient, ircAction);
             return;
         }
 
-        const potentialCodeText = trimCodeMessages(text);
-        if (potentialCodeText) {
-            ircAction.text = text = potentialCodeText;
-        }
-
-        let cacheBody = text;
+        let cacheBody = event.content.body;
```

Comment:
This changes the code so that we now cache the body, rather than `trimCodeMessages(body)`, which seems like a potentially dangerous change?